nerves_toolchain_aarch64_unknown_linux_gnueabi/mix.exs

```diff
@@ -1,42 +1,47 @@
+file = "nerves_toolchain_helpers.exs"
+file = if File.exists?(file), do: file, else: Path.join("..", file)
+Code.require_file(file)
+
 defmodule NervesToolchainAarch64UnknownLinuxGnueabi.Mixfile do
   use Mix.Project
 
-  @version Path.join(__DIR__, "VERSION")
-    |> File.read!
-    |> String.trim
+  @app     :nerves_toolchain_aarch64_unknown_linux_gnueabi
+  @version "1.0.0-dev"
 
   def project do
-    [app: :nerves_toolchain_aarch64_unknown_linux_gnueabi,
+    [app: @app,
      version: @version,
      elixir: "~> 1.4",
-     compilers: Mix.compilers ++ [:nerves_package],
+     compilers: [:nerves_package],
+     nerves_package: nerves_package(),
      description: description(),
      package: package(),
-     deps: deps()]
+     deps: deps(),
+     aliases: ["deps.precompile": ["nerves.env", "nerves.toolchain.helpers", "deps.precompile"]]]
   end
 
-  # Configuration for the OTP application
-  #
-  # Type "mix help compile.app" for more information
-  def application do
-    [env: [
-      target_tuple: :aarch64_unknown_linux_gnueabi
+  def nerves_package do
+   [type: :toolchain,
+    platform: Nerves.Toolchain.CTNG,
+    target_tuple: :aarch64_unknown_linux_gnueabi,
+    artifact_url: [
+      "https://github.com/nerves-project/toolchains/releases/download/v#{@version}/#{@app}-#{@version}.#{Nerves.Toolchain.host_platform()}-#{Nerves.Toolchain.host_arch()}.tar.xz"
+    ],
+    checksum: [
+      "defconfig"
     ]]
   end
 
-  # Dependencies can be Hex packages:
-  #
-  #   {:mydep, "~> 0.3.0"}
-  #
-  # Or git/path repositories:
-  #
-  #   {:mydep, git: "https://github.com/elixir-lang/mydep.git", tag: "0.1.0"}
-  #
-  # Type "mix help deps" for more examples and options
+  def application do
+    []
+  end
+
   defp deps do
     [
-      {:nerves, "~> 0.7"},
-      {:nerves_toolchain_ctng, "~> 1.1"}
+      #{:nerves, "~> 0.7", runtime: false},
+      {:nerves, github: "nerves-project/nerves", runtime: false},
```

Comment:
I can add a todo, but this requires what is in master for nerves. It should switch to a tag when its ready to be released.